src/main/java/org/opensha/sha/earthquake/rupForecastImpl/nshm23/erf/NSHM23_WUS_BranchAveragedERF.java

```diff
@@ -0,0 +1,83 @@
+package org.opensha.sha.earthquake.rupForecastImpl.nshm23.erf;
+
+import java.io.File;
+import java.io.IOException;
+
+import javax.swing.JOptionPane;
+
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.opensha.sha.earthquake.faultSysSolution.FaultSystemSolution;
+import org.opensha.sha.earthquake.faultSysSolution.erf.BaseFaultSystemSolutionERF;
+import org.opensha.sha.earthquake.rupForecastImpl.nshm23.util.NSHM23_Downloader;
+
+/**
+ * USGS 2023 NSHM ERF, Western U.S., Branch Averaged ERF
+ */
+public class NSHM23_WUS_BranchAveragedERF extends BaseFaultSystemSolutionERF {
+
+	private static final long serialVersionUID = 277613161331416141L;
+	private static final String MODEL = "WUS_branch_averaged_gridded_simplified";
+	public static final String NAME = "NSHM23 WUS Branch Avg ERF";
```

Comment:
I realized that we should specify that this ERF doesn't include the Cascadia subduction zone. Please rename to:

"NSHM23-WUS (excl. Cascadia) Branch Avg ERF

Reply:
SGTM, done!